Northstar.CustomServers/mod/scripts/vscripts/mp/_bleedout.gnut

```diff
@@ -36,77 +52,118 @@ void function Bleedout_OnClientConnected( entity player )
 	file.isBleeding[ player ] <- false
 	file.IsGettingFirstAidFrom[ player ] <- null
 	file.lastAttacker[ player ] <- svGlobal.worldspawn
+	file.lastAttackerWeapon[ player ] <- null
 }
 
 void function Bleedout_OnClientDisconnected( entity player )
 {
+	/* unhandled deleting( for players may already been destroyed ), no need for now
 	delete file.isBleeding[ player ]
 	delete file.IsGettingFirstAidFrom[ player ]
 	delete file.lastAttacker[ player ]
+	*/
 }
 
-void function Bleedout_SetCallback_OnPlayerStartBleedout( void functionref(entity) callback )
+void function Bleedout_AddCallback_OnPlayerStartBleedout( void functionref(entity, entity, var) callback )
 {
-	file.Callback_OnPlayerStartBleedout = callback
+	file.Callbacks_OnPlayerStartBleedout.append( callback )
 }
 
-void function Bleedout_SetCallback_OnPlayerGiveFirstAid( void functionref(entity) callback )
+void function Bleedout_AddCallback_OnPlayerGiveFirstAid( void functionref(entity, entity) callback )
 {
-	file.Callback_OnPlayerGiveFirstAid = callback
+	file.Callbacks_OnPlayerGiveFirstAid.append( callback )
 }
 
-void function Bleedout_StartPlayerBleedout( entity player, entity attacker )
+bool function Bleedout_StartPlayerBleedout( entity player, entity attacker, var damageInfo ) // victim, attacker, damageInfo
 {
 	//if the player is already bleeding don't restart bleeding logic.
 	if ( file.isBleeding[ player ] )
-		return
+		return false
 
+	if ( Bleedout_GetDeathOnTeamBleedout() )
+	{
+		bool isLastPlayer = CheckForTeamBleedout( player )
+		if( isLastPlayer )
+		{
+			// player will just die, done in sh_bleedout_damage.gnut
+			//player.Die( DamageInfo_GetAttacker( damageInfo ), DamageInfo_GetAttacker( damageInfo ), { damageSourceId = DamageInfo_GetDamageSourceIdentifier( damageInfo ) } )
```

Comment:
Instead of commenting out code and leaving a comment about to which source file the logic got moved to, I'd just simply remove the line and put the information in the PR description.

Otherwise overtime there it accumulate lots of
```
// moved to example.gnut
//foo(bar)
```
leftover comments that are no longer relevant ^^

This isn't the only place, just one that I noticed right now.